cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js

```diff
@@ -32,6 +33,30 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
         },
       );
     });
+    it('should render an SMP player that is ready to play media', () => {
+      cy.window().then(win => {
+        const media =
+          getBlockData('video', win.SIMORGH_DATA.pageData) ||
```

Comment:
Do we want this to work on AMP too?  

If so we need to do a couple of things:
- Move this code out of *canonicalOnly.js
- Get the data by making a request for the RAW json as per the `should render a media player` scenario above
https://github.com/bbc/simorgh/blob/9288c0eb8cf95e246729cb2785faf48b72001475/cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js#L20

Reply:
I had a go at this but it was taking a long time. For the moment, as discussed with @ryanmccombe this test is enough, as if the player is broken it should not work on both amp and canonical, so canonical will catch it.

I'll spend more time later making it work on amp, and also see if we can access the showCTA event which Jim said would be a good way of seeing if the player is working.